models/User/AbstractUser.php

```diff
@@ -124,7 +124,7 @@ public function getId()
     }
 
     /**
-     * @param int $id
+     * @param int|null $id
```

Comment:
I think we should not allow `null` here (in the future) ... thus not adding `null` to the type hints.